src/Core/Providers/ConsoleServiceProvider.php

```diff
@@ -542,6 +544,18 @@ protected function registerWidgetMakeCommand($abstract)
         });
     }
 
+    /**
+     * Register the make:request command.
+     *
+     * @param string $abstract
+     */
+    protected function registerRequestMakeCommand($abstract)
```

Comment:
The same goes for the declared method, the `registerRequestMakeCommand` should not be the last method call. Methods are also organized in alphabetical order in the file. Thank you.

Reply:
yes, sure.